GCRCatalogs/catalog_configs/dc2_metacal_griz_run2.1i_dr1b_tract2897.yaml

```diff
@@ -0,0 +1,9 @@
+subclass_name: dc2_metacal.DC2MetacalCatalog
+base_dir: /global/homes/f/flanusse/scratch/object_mcal_catalog_coaddv1_dr1b
+schema_filename: schema.yaml
+filename_pattern: 'metacal_tract_2897.parquet$'
+description: DC2 Run 2.1i coadd-v1b Metacal Catalog (griz)
+creators: ['Francois Lanusse', 'Johann Cohen-Tanugi']
+bands: 'griz'
+apply_metacal_test3_fix: true
+
```

Comment:
Better way to write this config is

```yaml
base_on: dc2_metacal_griz_run2.1i_dr1b.yaml
filename_pattern: 'metacal_tract_2897.parquet$'
```

In fact, if this is just used in the composite config, you don't even this standalone config at all. See below.